sources/packages/api/src-sql/StudentFiles/Add-meta-data-and-file-origin.sql

```diff
@@ -0,0 +1,23 @@
+-- set default
+ALTER TABLE
+  sims.student_files
+ALTER COLUMN
+  group_name
+SET
+  DEFAULT 'Student Temporary File';
```

Comment:
I was assuming that this would be default value while submitting the files for a student account, what means that it will be set in the file uploader component on form.io definition.
The AC states the below.
Default value of group_name while uploading **student account files** = Student Temporary File

Reply:
mmm, okay, i actually made it as default value in DB and in form 
![image](https://user-images.githubusercontent.com/77353155/156262034-eeacbbb8-631c-456a-b4de-0c23b88e2754.png)
. ya updating

Reply:
Cool, form.io is fine. Thanks for doing the change.